src/InfoBoxes/Content/Type.hpp

```diff
@@ -104,6 +104,7 @@ namespace InfoBoxFactory
     e_Alternate_1_Name, /* Displays name and bearing to the best alternate landing location */
     e_Alternate_2_Name, /* Displays name and bearing to the second alternate landing location */
     e_Alternate_1_GR, /* Geometric gradient to the arrival height above the best alternate. This is not adjusted for total energy */
+    e_Alternate_2_GR, /* Geometric gradient to the arrival height above the second alternate. This is not adjusted for total energy */
```

Comment:
Inserting this in the middle breaks existing configuration files

Reply:
Sorry about that I didn't noticed this side effect. This line is now moved before `e_NUM_TYPES`